homeassistant/components/remootio/exceptions.py

```diff
@@ -0,0 +1,10 @@
+"""Exceptions for the Remootio integration."""
```

Comment:
These are only used in config_flow btw. I think we can move this to there

Reply:
The two exceptions will be raised in `utils.py` by the private methods `_check_sensor_installed` and `_check_api_version` which will be invoked by the public methods `get_serial_number` and `create_client`.